### PR TITLE
prefix all classnames with 'rt-'

### LINKED ISSUE #29: Add class name prefixes to fix potential style leaks
As it currently stands we are not prefixing our classNames e.g. we have `.sidebar`
which means consumers CSS is likely to leak into the timeline.

I suggest we prefix all our classNames with `rt-` to avoid this.

### PR DESCRIPTION
**Pull request by rbrtsmith**
Fixes #29

### CODE REVIEW

File: src/components/Controls/Toggle.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 
 const Toggle = ({ toggleOpen, isOpen }) =>
   <button
-    className={`controls__button controls__button--toggle ${isOpen ? 'controls__button--toggle-close' : 'controls__button--toggle-open'}`}
+    className={`rt-controls__button r-tcontrols__button--toggle ${isOpen ? 'rt-controls__button--toggle-close' : 'rt-controls__button--toggle-open'}`}
```
Comment (atkinchris) [inline src/components/Controls/Toggle.jsx]: `r-tcontrols` -> `rt-controls`

File: src/scss/components/_track-keys.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+.rt-track-keys {}
+
 .track-keys {
```
Comment (atkinchris) [inline src/scss/components/_track-keys.scss]: Should this have been renamed?

File: src/scss/components/_track.scss [hunk 3]
```diff
@@ -1,15 +1,13 @@
-.track {
+.rt-track {}
```
Comment (atkinchris) [inline src/scss/components/_track.scss]: Do we need this empty declaration?

File: src/components/Controls/Toggle.jsx (hunk 1 above)
Comment (rbrtsmith) [inline src/components/Controls/Toggle.jsx]: Well spotted

**Pull request merged**